### PR TITLE
Change color of X logo and make X button work.

### PR DESCRIPTION
**Pull request by Jontes-Tech**

### CODE REVIEW

**Comment by lucemans** [comment]
Preferrably use `<FaTwitterX />` or whatver its called in the new react-icons

**Comment by Jontes-Tech** [comment]
Fixed

File: blog/src/components/navbar/HamburgerMenu.tsx [hunk 1]
```diff
@@ -16,7 +18,7 @@ const links = [
 
 const socials = [
     {
-        icon: <Image src={xLogo} className="w-5" alt="text-neutral-400" />,
+        icon: <RiTwitterXFill />,
```
Comment (svemat01) [inline blog/src/components/navbar/HamburgerMenu.tsx:21]: Any particular reason for using remix icons instead of font awesome?

**Review by lucemans — APPROVED** [review]
LGTM

**Pull request merged**